server/test/unit/vitalsTest/checkForInternalProblemsTest.js

```diff
@@ -0,0 +1,215 @@
+/* eslint-disable no-plusplus */
```

Comment:
Why did you disable the `no-plusplus` rule?

Reply:
I wanted to freely use ++ when I was writing my for loops for my tests 😄 I can definitely put it back though!